tests/providers/amazon/aws/operators/test_s3_file_transform.py

```diff
@@ -34,8 +33,8 @@
 from airflow.providers.amazon.aws.operators.s3 import S3FileTransformOperator
 
 
-class TestS3FileTransformOperator(unittest.TestCase):
-    def setUp(self):
+class TestS3FileTransformOperator:
+    def setup_method(self):
```

Comment:
Just optional suggestion (and reminder) because we could do it later by separate PR, when actually refactor some tests.

We manually create here temporary files and remove it in teardown_method, it could be replaced by new fixture which use `tmp_path_factory` builtin pytest fixture

https://github.com/apache/airflow/pull/27970/files#r1034288640

Reply:
I think this is best be done in a separate PR. I was also considering modifying this test, as it is actually untestable under MacOS, triggering an error: https://stackoverflow.com/questions/32115715/os-mknod-fails-on-macos
Thanks for the heads up!